src/introduction/introduction.tex

```diff
@@ -2,27 +2,51 @@
-One reason that chess has become more popular online, is that using the popular web apps like \url{chess.com} or
+perfected their craft, this has been done through intuitive design and the addition of extra features in the apps.
+One reason that chess has become more popular online, is that using popular web apps like \url{chess.com} or
 \url{lichess.org} allows the player to use a powerful tool to aid in their learning journey, that is chess engines.
 
+We believe that the utilization of chess engines is not fully satisfied yet, as engines are reserved for single player
+applications.
+
+% textidote: ignore begin
+\begin{figure}[htb]
+    \centering
+    \includegraphics[width=1\textwidth]{overview}
+    \caption{Understanding the project}\label{fig:project-overview}
+\end{figure}
+% textidote: ignore end
+
 Chess engine popularity has risen greatly in the past years and have been developed to a point that they
 outperform humans.
 These tools are essential to understanding the complexity of the end- and mid-game strategies of chess.
+This makes engines an optimal tool for teaching players the intricacies of chess which is exactly what
+differentiates the average chess players from masters.
+
+
 Chess encapsulates the spirit of a game that is easy to learn, but hard to master.
+That is why those who wish to become skilled in chess seek help from other masters in the form of a coach.
+Coaching is a well respected form of learning, but it has flaws that place barriers in terms of who can get this form of
+help.
+
+The biggest obstacle coaches have is the monetary aspect as most players do not have the money to afford a chess coach
+for extended periods of time.
+Coaches also need to have allocated time for their lessons which mean that it makes it more difficult to have the
+coaching sessions.
+
 We will be focusing on the learning aspect of chess by making a chess web app, that allows two players to play against
 each other with a coach.
-This coach will be a chess engine, which will allow both players to learn while playing in a multiplayer environment.
+This coach will be a chess engine, which will allow both players to learn while playing in a multiplayer environment, as
+seen in Figure~\ref{fig:project-overview}.
+
+
 
 Through this approach we touch a niche that lies between the social aspect of a player vs player chess game and the
 great learning tools that are made to study chess and its complexities individually.
 The product we are suggesting would then take the best of both worlds, by targeting the social aspects of the game while
 still providing useful information to the player on how to improve their chess capabilities.
-We go into more detail in Section~\ref{sec:state-of-the-art}.
+We also solve the two biggest problems with a real coach, as the proposed solution would be free and as the coach is an
+engine it is also available as long as the server is online.
```

Comment:
I dont think we should be concluding in the introduction. Maybe delete these lines

Reply:
I still think this should be here, but I agree that is is written as a conclusion what about something like this:
Making use of engines and online services can also adress the two main pain points of a real coach

Reply:
Lets go with that yeah